modules/core/src/alloy/UrlFormName.java

```diff
@@ -0,0 +1,34 @@
+/* Copyright 2022 Disney Streaming
+ *
+ * Licensed under the Tomorrow Open Source Technology License, Version 1.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    https://disneystreaming.github.io/TOST-1.0.txt
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package alloy;
+
+import software.amazon.smithy.model.node.Node;
+import software.amazon.smithy.model.shapes.ShapeId;
+import software.amazon.smithy.model.traits.AnnotationTrait;
+
+public final class UrlFormName extends AnnotationTrait {
```

Comment:
an annotation trait is a trait that is just an empty structure. That is not the case here : the POJO has to be able to hold the name value. It should therefore be analogous to `case class UrlFormName(name: String)` in Scala

Reply:
Moreover, the name of this class should have the `Trait` suffix, by convention